components/language-chooser/react/language-chooser-react-mui/src/LanguageChooser.tsx

```diff
@@ -275,12 +271,21 @@ export const LanguageChooser: React.FunctionComponent<ILanguageChooserProps> = (
                 lp.selectedLanguage?.iso639_3_code
               );
               return (
-                <div
+                <ScrollingHelper
                   key={index}
                   // We use this ref to scroll the initially selected language card into view
                   ref={
                     isSelectedLanguageCard ? selectedLanguageCardRef : undefined
                   }
+                  onMount={() => {
+                    if (
+                      props.initialSelectionLanguageTag &&
+                      isSelectedLanguageCard
+                    ) {
+                      // This should only run once
+                      setInitialScrollingNeeded(true);
```

Comment:
I'm don't understand why this only ever runs once, even if you change the search string and then select a card that wasn't there before. But it works! Running only once is the desired behavior